sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -5995,7 +5998,7 @@ public void testTimeseriesUsingTimeFloorWithTimeShift() throws Exception
                         .setVirtualColumns(
                             expressionVirtualColumn(
                                 "v0",
-                                "timestamp_floor(timestamp_shift(\"__time\",'P1D',-1),'P1M',null,'UTC')",
+                                "timestamp_floor(timestamp_shift(\"__time\",'P1D',-1,'UTC'),'P1M',null,'UTC')",
```

Comment:
Should we add a test with non UTC timezone as well?

Reply:
I added one in ExpressionsTest.

Reply:
Thanks.